package.json

```diff
@@ -12,5 +14,12 @@
   "bugs": {
     "url": "https://github.com/topcoder-platform/plat-interface-definition/issues"
   },
-  "homepage": "https://github.com/topcoder-platform/plat-interface-definition#readme"
+  "homepage": "https://github.com/topcoder-platform/plat-interface-definition#readme",
+  "devDependencies": {
+    "@commitlint/config-conventional": "^17.4.0",
+    "@commitlint/config-lerna-scopes": "^17.4.0",
```

Comment:
lerna-scopes aren't needed since we aren't using lerna in this repo. We likely need to add something in commitlint config `rules` section to "whitelist" list of scopes